clients/cloud/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,45 @@
+group = "io.confluent.ccloud"
+
+plugins {
+  java
+  maven
+  kotlin("jvm") version "1.3.11"
+}
+
+java {
+  sourceCompatibility = JavaVersion.VERSION_1_8
+  targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+dependencies {
+  compile("org.jetbrains.kotlin:kotlin-stdlib")
+  compile("org.jetbrains.kotlin:kotlin-reflect")
+
+  compile("org.apache.kafka:kafka-clients:2.1.0")
+  compile("org.apache.kafka:kafka-streams:2.1.0")
+  compile("org.apache.kafka:connect-runtime:2.1.0")
+  compile("io.confluent:kafka-json-serializer:5.0.1")
+  compile("org.slf4j:slf4j-api:1.7.6")
+  compile("org.slf4j:slf4j-log4j12:1.7.6")
+  compile("com.fasterxml.jackson.core:jackson-databind:[2.8.11.1,)")
+  //compile ("com.fasterxml.jackson.module:jackson-module-kotlin:[2.8.11.1,)")
+  compile("com.google.code.gson:gson:2.2.4")
+}
+
+repositories {
+  jcenter()
+
+  maven(url = "http://packages.confluent.io/maven/")
+  maven(url = "http://repo.maven.apache.org/maven2")
+}
+
+val configPath: String by project
+val topic: String by project
+val mainClass: String by project
+
```

Comment:
Sorry for posting this late:
I think you need to set jvmTarget to 1.8. Default is 1.6
See here: https://kotlinlang.org/docs/reference/using-gradle.html#attributes-specific-for-jvm

Reply:
it's there

```
java {
  sourceCompatibility = JavaVersion.VERSION_1_8
  targetCompatibility = JavaVersion.VERSION_1_8
}
```

Reply:
done.
Thank you